documentation/src/pages/recipes/data/recipes/swagger-to-markdown.yaml

```diff
@@ -0,0 +1,23 @@
+version: "1.0.0"
+title: Swagger to Markdown
```

Comment:
While I'm not opposed to recipes like this, there are far better, more thorough open source tools that exist for this exact reason; I'm not sure this provides more value to the community than those other tools. Let's remove this from your pull request and revisit this in the future.

Reply:
https://block.github.io/goose/recipes/detail?id=dependency-updater